alert-common/src/main/java/com/synopsys/integration/alert/common/channel/issuetracker/config/IssueTrackerServiceConfig.java

```diff
@@ -0,0 +1,26 @@
+/**
+ * alert-common
+ *
+ * Copyright (c) 2020 Synopsys, Inc.
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package com.synopsys.integration.alert.common.channel.issuetracker.config;
+
+public interface IssueTrackerServiceConfig {
```

Comment:
Is this exclusively a marker interface? or are there any common methods we can add here?

Reply:
Just a Marker interface with AzureBoards we may be able to change it to have common methods much like the Provider class when we started supporting multiple providers.